src/openforms/js/components/admin/form_design/registrations/zgw/fields/RoltypeFields.js

```diff
@@ -133,7 +122,35 @@ const RoltypeFields = ({catalogueUrl = ''}) => {
           }
           noManageChildProps
         >
-          <PartnersRoltypeSelect
+          <RoltypeSelect
+            roltypeTypeName="partnersRoltype"
+            roltypes={roleTypes}
+            loading={loading}
+            zgwApiGroup={zgwApiGroup}
+            caseTypeIdentification={caseTypeIdentification}
+          />
+        </Field>
+      </FormRow>
+      <FormRow>
+        <Field
+          name="childrenRoltype"
+          label={
+            <FormattedMessage
+              description="ZGW-APIs registration options 'childrenRoltype' label"
+              defaultMessage="Children roltype"
+            />
+          }
+          helpText={
+            <FormattedMessage
+              description="ZGW-APIs registration options 'childrenRoltype' help text"
+              defaultMessage={`Description (omschrijving) of the ROLTYPE to use for citizens
+                filling in a form with children.`}
+            />
+          }
+          noManageChildProps
+        >
+          <RoltypeSelect
```

Comment:
Should both of the `RolTypeSelect` fields have the same `roleTypes`?

Reply:
The `rolTypes` value is coming from the API call we make a bit above. The call is the same for all the components because it's based on the `zgwApiGroup`, `catalogueUrl`, and `caseTypeIdentification`. We already had this here for the `medewerkerRoltype` and the `partnersRoltype` so I guess there is no change. 

We had discussed this last time as well:
https://github.com/open-formulieren/open-forms/pull/5386#discussion_r2190183400